doc/sphinx/changes.rst

```diff
@@ -455,7 +455,7 @@ Tactic language
   (`#11882 <https://github.com/coq/coq/pull/11882>`_,
   by Hugo Herbelin).
 - **Added:**
-  Ltac2 notations for reductions in terms: :n:`eval @red_expr in @ltac2_term`
+  Ltac2 notations for reductions in terms: :n:`eval @red_expr in @ltac_expr`
```

Comment:
Correct?

```suggestion
  Ltac2 notations for reductions in terms: :n:`eval @red_expr in @ltac2_expr`
```

Reply:
Actually, neither.  The grammar has 

```
constr_eval: [
| "eval" red_expr "in" Constr.constr
```

so it should be @constr AKA @term.  I'll fix.